internal/pkg/build/metadata.go

```diff
@@ -341,6 +341,12 @@ func addBuildLabels(labels map[string]string, b *types.Bundle) error {
 		}
 	}
 
+	// Digest of image
+	if b.Opts.Tag != "" && b.Opts.Digest != "" {
+		labels["org.label-schema.image-tag"] = b.Opts.Tag
+		labels["org.label-schema.image-digest"] = b.Opts.Digest
```

Comment:
Are these label names made up or are they part of some standard?

Reply:
They are made up, or well the "tag" and "digest" is part of the Docker format

Reply:
Note that both words are highly ambiguous, but here they are both clearly defined.

"Tag" is the **full** reference name, and "Digest" is the **full** image digest (with algorithm).

Reply:
I don't think it's OK to make up org.label-schema namespace labels, because those are defined by `org.label-schema`.  It should be allowed to make `org.apptainer` labels.  I see the `org.label-schema` space is now [deprecated](http://label-schema.org/rc1/) and replaced by [`org.opencontainers.image`](https://github.com/opencontainers/image-spec/blob/main/annotations.md).  That says that that namespace MUST NOT be used by extensions.  On the other hand it looks like singularity and apptainer have already added on to `org.label-schema.usage` with sublabels.

Reply:
I suppose that we could also add `org.opencontainers.image.created` in addition to the old build-date?

But there doesn't seem to exist any field for the full `platform`, I can't even find any reference to build-arch